awsbatch-cli/setup.py

```diff
@@ -22,7 +22,7 @@ def readme():
 
 VERSION = "1.4.0"
 REQUIRES = [
-    "setuptools<70.0.0",
```

Comment:
What was the reason we needed to pin this deps?
Are we sure the reason is not valid anymore and the pin removal does not need depend on further changes?